### PR TITLE
Add Lambda specific page to Graviton page

### PR DESCRIPTION
**Pull request by julianwood**
Add Lambda specific page to Graviton page.

Description of how Lambda on Graviton2 works with demos to help customers migrate their functions. 

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,29 +3,21 @@
...
-* [Building for Graviton](#building-for-graviton-and-graviton2)
-* [Optimizing for Graviton](optimizing.md)
-* [Taking advantage of Arm Advanced SIMD instructions](SIMD_and_vectorization.md)
-* [Recent software updates relevant to Graviton](#recent-software-updates-relevant-to-graviton)
-* Language-specific considerations
-	* [C/C++](c-c++.md)
-	* [Go](golang.md)
-	* [Java](java.md)
-	* [.NET](dotnet.md)
-	* [Python](python.md)
```
Comment (AGSaidi) [inline README.md:16]: this chase seems to have done more than just add a Lambda page?

Comment (julianwood) [reply README.md:16]: This adds a Lambda page as well as some code samples to show customers how to migrate their functions and how to test Lambda on x86 and arm. Customers can use the code to deploy a sample Lambda function and compare performance.

Comment (AGSaidi) [reply README.md:16]: My point is that the change significantly changed the table of contents beyond adding the lambda section.

Comment (julianwood) [reply README.md:16]: Ah, I see, yes, this shouldn't have been as big a change to the TOC, I think I've been out of sync with other updates. I'll close PR and reopen with a cleaner one.

**Pull request closed without merging**